From 3c79baf6322e9ab8e5f12775af89906454f4a804 Mon Sep 17 00:00:00 2001
From: Chen Chen <cncn@google.com>
Date: Tue, 5 Oct 2021 17:46:02 -0700
Subject: [PATCH 1/2] BluetoothSecurity: Add BLUETOOTH_PRIVILEGED permission
 for pairing dialog

Bug: 194300867
Test: Build

Change-Id: I5a496df50550ea7ee6986c960e28ae3e4a056b4b
(cherry picked from commit e5ccfba9962f22d05cbf9cc6097ef7b74938ecf0)
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index c32309a0a1..1958bfdd98 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -2018,6 +2018,7 @@
         </activity>
 
         <activity android:name=".bluetooth.BluetoothPairingDialog"
+                  android:permission="android.permission.BLUETOOTH_PRIVILEGED"
                   android:excludeFromRecents="true"
                   android:windowSoftInputMode="stateVisible|adjustResize"
                   android:theme="@style/Theme.AlertDialog"
-- 
2.25.1

